ui/src/widgets/ui-gauge/types/UIGaugeDial.vue

```diff
@@ -511,10 +522,21 @@ export default {
 
 /* Size Overrides */
 
+/* xxs */
+/* hide gauge textual contents as they are not visible to naked eye in this grid size */
+.nrdb-ui-gauge-size-xxs .nrdb-ui-gauge-value,
+.nrdb-ui-gauge-size-xxs #limits {
+    display: none;
+}
+
 /* xs */
 .nrdb-ui-gauge-size-xs .nrdb-ui-gauge-value span {
-    font-size: 1.5rem;
-    line-height: 1.75rem;
+    font-size: min(1.5rem,max(2cqmin,1.2rem));
+    line-height: min(1.5rem,max(2cqmin,1.5rem));
+}
+.nrdb-ui-gauge-size-xs .nrdb-ui-gauge-value {
+    font-size: min(.75rem,max(2cqmin,.5rem));
+    line-height: min(.825rem,max(2cqmin,.65rem));
```

Comment:
You're using `cqmin`, but never actually declare where your parent container is?

Reply:
Thanks for pointing that out! `cqmin` automatically inherits the parent container's dimensions for its calculations, so I didn’t explicitly define a parent container

Reply:
> To use container queries, you need to declare a containment context on an element so that the browser knows you might want to query the dimensions of this container later. To do this, use the [container-type](https://developer.mozilla.org/en-US/docs/Web/CSS/container-type) property with a value of size, inline-size, or normal.

https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_containment/Container_queries#using_container_queries

Reply:
You're correct. we need to declare where the parent container is the type of the container. I fixed it